assets/app.js

```diff
@@ -1530,6 +1491,27 @@ new Vue({
         switchRemoveMetatag: function(){
             this.remove_mode_metatag = !this.remove_mode_metatag
         },
+        switchTranslationStatus: function(type, cell){
+            proceed = cell.row['translation_requested'] == 1
+                ? confirm('Have you translated in all language of "' + cell.row[this.editorLanguage] + '"? Please only proceed if you are sure about it.')
+                : confirm('Do you want to request for translation of "' + cell.row[this.editorLanguage] + '"? Please only proceed if you are sure about it.')
+
+            if (proceed) {
```

Comment:
you could have done: `if(proceed && hasApi) {}` so it'd have one less branch and less complexity?

Reply:
yes, @cyberhck is right. You just test for the two booleans once.